### PR TITLE
add beers controller and view

### PR DESCRIPTION
**Pull request by mattphoto**

### CODE REVIEW

File: templates/views/beers.html [hunk 1]
```diff
@@ -1,23 +1,81 @@
...
+</div>
+
+<div class="col-md-12" align="center">
+    {% for beer in data.allBeers %}
+    {% if loop.index0%6 == 0 %}
+    <div class="col-md-12" align="center">
+      {% endif %}
+      {% if beer.beerImage.url != nil %}
+      <div class="col-md-2" align="center">
+        <img src={{beer.beerImage.url}} class="img-thumbnail">
```
Comment (lizTheDeveloper) [inline templates/views/beers.html:54]: Put quotes around the src attribute- 
src="{{beer.beerImage.url}}"

File: templates/views/beers.html [hunk 2]
```diff
@@ -1,23 +1,81 @@
...
+</div>
+
+<div class="col-md-12" align="center">
+    {% for featuredBeer in filters.featuredBeers %}
+    {% if loop.index0%2 == 0 %}
+    <div class="col-md-12" align="center">
+      {% endif %}
+      {% if featuredBeer.beerImage.url != nil %}
+      <div class="col-md-6" align="center">
+        <img src={{featuredBeer.beerImage.url}} class="img-thumbnail">
```
Comment (lizTheDeveloper) [inline templates/views/beers.html:26]: Quotes around src- src="{{featuredBeer.beerImage.url}}"

File: templates/views/beers.html (hunk 1 above)
Comment (lizTheDeveloper) [inline templates/views/beers.html:54]: Also add a link here to the actual profile of this beer

File: templates/views/beers.html [hunk 3]
```diff
@@ -1,23 +1,81 @@
...
+    {% if (loop.index0+1)%6 == 0 %}
+</div>
+{% endif %}
+{% endfor %}
+</div>   
+
+<div class="col-md-12" align="center">
+    <div class="container">
+      <p>Featured beers</p>
+      <button class="btn btn-primary">Try these beers</button>
```
Comment (lizTheDeveloper) [inline templates/views/beers.html:74]: What does this button do? Is it in the spec?

File: templates/views/beers.html [hunk 4]
```diff
@@ -1,23 +1,81 @@
...
+      <div class="col-md-2" align="center">
+        <img src={{beer.beerImage.url}} class="img-thumbnail">
+        <br>
+        {{beer.beerName.first}} {{beer.beerName.last}}
+    </div>
+    {% else %}
+    <div class="col-md-2" align="center">
+        <img src="http://cdn3.tapcdn.com/images/source/8f/4b/4c/89/bac6aa15e59b62c09a783b79.png" class="img-thumbnail">
+        <br>
+        {{beer.beerName.first}} {{beer.beerName.last}}
```
Comment (lizTheDeveloper) [inline templates/views/beers.html:62]: Beer should have only one name, not a first and last name (beers are not people.)

File: routes/index.js [hunk 5]
```diff
@@ -42,6 +42,7 @@ exports = module.exports = function(app) {
 	app.get('/about', routes.views.about);
 	app.get('/brewers', routes.views.brewers);
 	app.get('/beers', routes.views.beers);
+	app.get('/beerProfile', routes.views.beers);
```
Comment (lizTheDeveloper) [inline routes/index.js:45]: This beerProfile should accept a beer slug as a parameter, and present a profile page for that beer.

File: routes/views/beerProfile.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        locals = res.locals;
+
+    locals.section = 'beerProfile';
+    locals.data = {
+        allBeers: []
+    };
+
+    view.on('init', function(next) {
+
+        var q = keystone.list('Beers').model.find().sort('-beerName')
```
Comment (lizTheDeveloper) [inline routes/views/beerProfile.js:15]: You should take the slug in here and find one- look at Mongoose's findOne method.

File: routes/views/beerProfile.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+        q.exec(function(err,results) {
+
+            locals.data.allBeers = results;
```
Comment (lizTheDeveloper) [inline routes/views/beerProfile.js:19]: and this should bring in only one beer

File: templates/views/beerProfile.html [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+{% block intro %}
+
+{% endblock %}
+
+{% block content %}
+
+</script>
+<div class="bodyToLoad">
+	{% for beer in data.allBeers %}
```
Comment (lizTheDeveloper) [inline templates/views/beerProfile.html:13]: Profile should be for a single beer, not all beers.

File: templates/views/beerProfile.html [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+		<br />
+		<br />
+		<br />
+
+
+		<div class="row">
+			<div class="col-md-6">
+				<div class="col-md-12">
+					<h4>Connect with the Brewer</h4>
```
Comment (lizTheDeveloper) [inline templates/views/beerProfile.html:64]: This should pull in the brewer data as they're associated, but we can put that in another pull request.

File: templates/views/beers.html (hunk 3 above)
Comment (mattphoto) [inline templates/views/beers.html:74]: It's in the UXpin & in Anders layout. I believe it goes to the subscription page.
